src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -23,6 +23,14 @@ namespace Microsoft.PowerShell.Commands
     [OutputType(typeof(PSModuleInfo))]
     public sealed class TestModuleManifestCommand : ModuleCmdletBase
     {
+        /// <summary>
+        /// Creates an instance of the Test-ModuleManifest command
+        /// </summary>
+        public TestModuleManifestCommand()
+        {
+            BaseSkipEditionCheck = true;
```

Comment:
For my education, can you please explain why this is needed?

Reply:
This calls `ModuleCmdletBase.LoadModuleManifest()` and throws if `BaseSkipEditionCheck` is not set.

On the basis that `Test-ModuleManifest` should never reject a manifest based on incompatibility (only manifest invalidity), I just set it to true.

However, if `Test-ModuleManifest` should have a more sophisticated functionality, let me know I and I'll change this accordingly.

Reply:
OK, can you please add comment here explain why it's set to true here?